proposals/lip-0051.md

```diff
@@ -123,71 +128,166 @@ The following constants are used throughout the document:
+
+| Name                                 | Type    | Value  | Description |
+|--------------------------------------|---------|--------|-------------|
+| **Names**                         |         |        |             |
+| `EVENT_NAME_TRANSFER`                | string  | "transfer"    | Used for events emitted during token transfers. |
+| `EVENT_NAME_TRANSFER_CROSS_CHAIN`    | string  | "transferCrossChain"    | Used for events emitted during cross-chain token transfers. |
+| `EVENT_NAME_CCM_TRANSFER`            | string  | "ccmTransfer"    | Used for events emitted during execution of cross-chain token transfer messages. |
+| `EVENT_NAME_MINT`                    | string  | "mint"    | Used for events emitted during calls to the [mint](#mint) function. |
+| `EVENT_NAME_BURN`                    | string  | "burn"  | Used for events emitted during calls to the [burn](#burn) function. |
+| `EVENT_NAME_LOCK`                    | string  | "lock"    | Used for events emitted during calls to the [lock](#lock) function. |
+| `EVENT_NAME_UNLOCK`                  | string  | "unlock"    | Used for events emitted during calls to the [unlock](#unlock) function. |
+| `EVENT_NAME_INITIALIZE_TOKEN`        | string  | TBD    | Used for events emitted during calls to the [initializeToken](#initializetoken-1) function. |
+| `EVENT_NAME_INITIALIZE_USER_STORE`   | string  | TBD    | Used for events emitted during initialization of user substore entry.|
+| `EVENT_NAME_INITIALIZE_ESCROW_STORE` | string  | TBD    | Used for events emitted during initialization of escrow substore entry.|
+| `EVENT_NAME_RECOVER`                 | string  | "recover"    | Used for events emitted during calls to the [recover](#recover) function. |
+| `EVENT_NAME_BEFORE_CCC_EXECUTION`    | string  | "beforeCCCExecution"  | Used for events emitted during calls to the [beforeCrossChainCommandExecution](#beforecrosschaincommandexecution) function. |
+| `EVENT_NAME_BEFORE_CCM_FORWARDING`   | string  | "beforeCCMForwarding"  | Used for events emitted during calls to the [beforeCrossChainMessageForwarding](#beforeCrossChainMessageForwarding) function. |
+| `EVENT_NAME_ALL_TOKENS_SUPPORTED`| string  | TBD    | Used for the [allTokensSupported](#alltokenssupported) event  |
+| `EVENT_NAME_ALL_TOKENS_SUPPORTED_REMOVED`| string  | TBD    | Used for the [allTokensSupportedRemoved](#alltokenssupportedremoved) event  |
+| `EVENT_NAME_ALL_TOKENS_FROM_CHAIN_SUPPORTED`| string  | TBD    | Used for the [allTokensFromChainSupported](#alltokensfromchainsupported) event  |
+| `EVENT_NAME_ALL_TOKENS_FROM_CHAIN_SUPPORTED_REMOVED`| string  | TBD    | Used for the [allTokensFromChainSupportedRemoved](#alltokensfromchainsupportedremoved) event  |
+| `EVENT_NAME_TOKEN_ID_SUPPORTED`      | string  | TBD    | Used for the [tokenIDSupported](#tokenidsupported) event  |
+| `EVENT_NAME_TOKEN_ID_SUPPORT_REMOVED`      | string  | TBD    | Used for the [tokenIDSupportRemoved](#tokenidsupportremoved) event  |
```

Comment:
I think since the last event is `EVENT_NAME_TOKEN_ID_SUPPORT_REMOVED`, `EVENT_NAME_ALL_TOKENS_SUPPORTED_REMOVED` and `EVENT_NAME_ALL_TOKENS_FROM_CHAIN_SUPPORTED_REMOVED` these should be `XXX_SUPPORT_REMOVED` event

Reply:
Good point. Fixed.